modules/hawkular-api-parent/hawkular-listener/src/main/java/org/hawkular/listener/bus/InventoryEventListener.java

```diff
@@ -696,14 +696,10 @@ private void handleResourceEvent(ResourceEvent event) {
                 }
                 case "URL": {
                     // common to members
-                    String feedId = r.getPath().ids().getFeedId();
-                    String url = event.getHeaders().get("url");
-                    //String resourceId = String.valueOf(MessageDigest.getInstance("MD5").digest(
-                    //        r.getId().getBytes("UTF-8")));
+                    // TODO: use the URL as needed, not the hash if and when we can get the actual URL from the event
                     String resourceId = r.getId();
-                    //String qualifiedResourceId = feedId + "/" + resourceId;
                     Map<String, String> memberContext = new HashMap<>(2);
-                    memberContext.put("resourceName", url);
+                    memberContext.put("resourceName", "URL [" + resourceId + "]"); // Should be the URL
```

Comment:
The actual URL is not supplied by inventory, so using the resourceId is a workaround until such time that we have the URL.